### ISSUE TITLE
Documentation hamburger button lacks visible focus indication

### ISSUE THREAD

**Issue by patrickhlauke** [twbs/bootstrap#30420]
The hamburger menu button in v4 and upcoming v5 documentation on small screen lacks a visible focus indication (e.g. when navigating by keyboard)

![v4 header with hamburger menu button](https://user-images.githubusercontent.com/895831/77048060-bd118980-69bd-11ea-91d0-a8ef46428e8e.png)

![v4 header with hamburger menu button](https://user-images.githubusercontent.com/895831/77048035-b08d3100-69bd-11ea-8cb6-79cf1d892e2f.png)

incidentally, this would be the sort of thing that would ideally, in future, be tackled using `:focus-visible` (as you'd want some sort of focus outline, but ideally only when navigating with keyboard and not after clicking with mouse/tapping on touchscreen)

xref https://github.com/twbs/bootstrap/pull/30073#issuecomment-601054410

**Comment by TorRydberg** [comment]
Hi @patrickhlauke, I have a fix for this. Can I commit my code?
![Bootstrap](https://user-images.githubusercontent.com/50177532/78816297-88cb3080-799f-11ea-9043-3da11d76af53.PNG)

**Comment by ffoodd** [comment]
@TorRydberg please do 🙂

**Comment by MartijnCuppens** [comment]
There seem to be an indication in v5 already: https://twbs-bootstrap.netlify.com/docs/4.3/getting-started/introduction/

![image](https://user-images.githubusercontent.com/11559216/78863893-e6f92180-7a3a-11ea-95fc-f7622bba9a09.png)

**Comment by ffoodd** [comment]
Confirmed, coming from `.btn:focus`. And the related PR is using `border`, which moves the button around on focus…

Don't know what happened here, but I suggest to close both issue and PR. @patrickhlauke Could you check this again?

**Comment by MartijnCuppens** [comment]
> but I suggest to close both issue and PR

This is still an issue in v4

**Comment by MartijnCuppens** [comment]
Gonna close this, this was fixed by https://github.com/twbs/bootstrap/pull/30036. Opened #30542 to remove the redundant `.btn-link` class which has no effect.
